.github/PULL_REQUEST_TEMPLATE/bug.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Pull Request for a bug
+about: Use this template for bugs
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+<!-- Thank you so much for your time taking to contribute, your work is appreciated! 😃 -->
+
+## Description
+<!-- A few sentences describing the overall goals of the pull request' s commits. -->
```

Comment:
I suggest to ask to describe the real problem AND the solution implemented.
Because the problem is often described in terms of functionalities. Here we should have e description of the real reason of the problem and the solution proposed. So it's clear to the reviewers and for future checks.

Reply:
the problem should be described in the issue